repo/packages/B/beta-tensorflow/1/config.json

```diff
@@ -0,0 +1,329 @@
+          "description": "Path to your TensorFlow job (assume job_url has already been downloaded/unpacked to the current directory). This path will be added to PYTHONPATH at runtime.",
+          "type": "string",
+          "default": ""
+        },
+        "job_name": {
+          "title": "Job Name",
+          "description": "Name of the module (without extension) whose main function should be executed.",
+          "type": "string",
+          "default": ""
+        },
+        "job_context": {
+          "title": "Job Context (Optional)",
+          "description": "A dictionary to pass to your job's main function (intended for run-time configuration options like batch_size, learning_rate, etc.).",
+          "type": "string",
+          "default": "{}"
+        },
+        "shared_filesystem": {
+          "title": "Shared Filesystem",
+          "description": "Distributed TensorFlow relies on a shared filesystem (GCS, HDFS, etc.) for summaries/checkpoints. Please set up one of these filesystems before using this package, and specify the location here (eg. 'gs://your/directory' or 'hdfs://namenode:port/your/directory').",
+          "type": "string",
+          "default": ""
+        },
+        "gcs_key": {
+          "title": "GCS Key",
+          "description": "Stringified contents of JSON key for Google Cloud Storage service account. Only use this field if use_gcs_key_secret is set to False.",
+          "type": "string",
+          "default": ""
+        },
+        "use_gcs_key_secret": {
+          "description": "A DC/OS secret called gcs_key exists and should be consumed by this framework.",
+          "type": "boolean",
+          "default": false
+        },
+        "use_tensorboard": {
+          "description": "Should the framework also launch an instance of Tensorboard? Make sure you have a shared filesystem set up first! You can still run Tensorboard locally if this is set to false",
+          "type": "boolean",
+          "default": false
+        },
+        "tf_image": {
+          "title": "TensorFlow Docker Image",
+          "description": "Name of the TensorFlow Docker Image in which Worker/PS tasks will be run (usr/img:version).",
+          "type": "string",
+          "default": "mesosphere/dcos-tensorflow:2.7.4-1.3.0"
+        },
+        "gpu_tf_image": {
+          "title": "TensorFlow GPU Docker Image",
+          "description": "Name of the TensorFlow Docker Image in which GPU Worker tasks will be run (usr/img:version).",
+          "type": "string",
+          "default": "mesosphere/dcos-tensorflow:2.7.4-1.3.0-gpu"
+        }
+      },
+      "required": [
+        "name",
+        "user",
+        "job_url",
+        "job_name",
+        "tf_image",
+        "gpu_tf_image"
+      ]
+    },
+    "parameter_server": {
+      "description": "TF Parameter Server configuration properties",
+      "type": "object",
+      "properties": {
+        "count": {
+          "title": "Node count",
```

Comment:
is is node or task count? the description seems slightly out of sync with name...

Reply:
Also, PS is spelled capital in the other occurrences below.

Reply:
I would not abbreviate anything. I.e. always write out TensorFlow, Parameter Server, etc.